src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./styles/style.scss"
+    >
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com"
```

Comment:
Still not fixed. You need to check and fix it everywhere
<img width="1064" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/c3821165-c94b-4eaa-a9b1-6df0f9b8a3d8">